recipes-kernel/linux/4.14/defconfigs/xenclient-dom0/defconfig

```diff
@@ -304,7 +304,15 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_MODULE_FORCE_UNLOAD=y
 CONFIG_MODVERSIONS=y
 CONFIG_MODULE_SRCVERSION_ALL=y
-# CONFIG_MODULE_SIG is not set
+CONFIG_MODULE_SIG=y
+CONFIG_MODULE_SIG_FORCE=y
```

Comment:
It is my understanding that enabling this will prevent loading an unsigned module.
Yet, with build [$1653](http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1653), unsigned modules are loaded in dom0  (e.g, `fbtap`, its recipe does not inherit `kernel-signing.bbclass`).

I must be missing something or the signing is not performed while `CONFIG_SIG_FORCE` is set.